src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -88,6 +88,10 @@ internal static class IDEDiagnosticIds
 
         public const string FormattingDiagnosticId = "IDE0055";
 
+        public const string ExpressionValueIsUnusedDiagnosticId = "IDE0056";
```

Comment:
>IDE0056 [](start = 67, length = 7)

I will fix the merge conflicts from already used diagnostics IDs when the PR is ready to merge.